src/styles/card.scss

```
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
```

Use variables for colors, for example:

```suggestion
  border: 1px solid $white-color;;
```